samples/server/petstore/jaxrs/jersey2/pom.xml

```diff
@@ -163,7 +163,7 @@
     <java.version>1.7</java.version>
     <maven.compiler.source>${java.version}</maven.compiler.source>
     <maven.compiler.target>${java.version}</maven.compiler.target>
-    <swagger-core-version>1.5.9</swagger-core-version>
+    <swagger-core-version>1.5.12</swagger-core-version>
```

Comment:
I've added back the dependency via https://github.com/swagger-api/swagger-codegen/pull/4881/commits/f15267c717f56803c518c118270c85c0ab473252